src/app.module.ts

```diff
@@ -3,9 +3,24 @@ import { AppController } from './app.controller';
 import { AppService } from './app.service';
 import { ConfigProjModule } from './config/config.module';
 import { UserModule } from './user/user.module';
+import { TaskModule } from './task/task.module';
+import { DynamooseModule } from 'nestjs-dynamoose';
 
 @Module({
-  imports: [ConfigProjModule, UserModule],
+  imports: [
+    ConfigProjModule,
+    DynamooseModule.forRoot({
+      aws: {
+        accessKeyId: process.env.AWS_ID,
+        secretAccessKey: process.env.AWS_SECRET,
+        region: process.env.AWS_REGION,
+      },
+      local: Boolean(process.env.AWS_LOCAL),
+      logger: false,
```

Comment:
for development purpose.. wouldn't be better for dev environment logger in true ? or it doesn't make sense ?

Reply:
Yes, i agree!